nds-org/ndslabs
# Update user documentation for V2

## Problem
Our documentation is either out-of-date or spread around in several places

Fixes #394 

## Approach
* feat: update Sphinx docs to include much of this information

![Screenshot 2023-12-20 at 7 36 38 PM](https://github.com/nds-org/ndslabs/assets/1413653/cf530487-220b-4025-a6c2-3a15c6d661dc)

## How to Test
1. Navigate to https://ndslabs.readthedocs.io to see the new version
     * Please point out any typos or confusing sections, and I can try to reword them

### TODOs
* More screenshots
* Fill in UserApp/AppSpec definitions
* Search for other TODOs within the code
